src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -670,7 +680,7 @@ export class ServerView extends DirMixin(PolymerElement) {
       this.selectedTab = 'connections';
       this.accessKeySortBy = 'name';
       /** The direction to sort: 1 == ascending, -1 == descending */
```

Comment:
Can you mention in the comment that 0 == no sorting?

Reply:
Done